content/modules/rest/pages/entities-api/update-entities.adoc

```diff
@@ -0,0 +1,404 @@
+
+In this example you will see how to update an `M:N` association between `Product` and `ProductTag`. Assuming that the stored Product before the update looks like this:
+
+[source, json]
+.Existing Product
+----
+{
+  "_entityName": "rstex11_Product",
+  "_instanceName": "123",
+  "id": "e1d586b4-aefb-2ee7-3b91-b07357b178ea",
+  "createdDate": "2021-03-05",
+  "createdBy": "admin",
+  "lastModifiedDate": "2021-03-05",
+  "price": 99.95,
+  "name": "Outback Power Remote Power System",
+  "version": 1,
+  "tags": [
+    {
+      "_entityName": "rstex11_ProductTag",
+      "_instanceName": "shiny",
+      "id": "333f3a20-c47b-4bc9-ba34-a72d2d815695",
+      "name": "shiny"
+    },
+    {
+      "_entityName": "rstex11_ProductTag",
+      "_instanceName": "great",
+      "id": "c4c028f0-fec1-7512-83cd-c17537d1f502",
+      "name": "great"
+    }
+  ]
+}
+----
+
+In this example you would like to change the tags of the Product in the following ways:
+
+* `great` should be removed from the tags.
+* `amazing` should be added to the tags.
+* The tag `shiny` should be kept.
+
+[source, http request]
+.Update Tag Associations Request
+----
+PUT http://localhost:8080/rest
+            /entities
+            /rstex11_Product
+            /e1d586b4-aefb-2ee7-3b91-b07357b178ea
+            ?responseFetchPlan=product-with-tags
+
+{
+  "name": "123",
+  "price": 99.95,
+  "tags": [
+    {
+      "id": "333f3a20-c47b-4bc9-ba34-a72d2d815695" // <1>
+    },
+    {
+      "id": "d6ab132e-a0bd-a624-c6ad-cc544e83c584" // <2>
+    }
+  ]
+}
+----
+<1> The ID of the product tag `shiny` is part of the request in order to keep the association.
+<2> The ID of the product tag `amazing` is newly added to the association.
+
+NOTE: As the ID of the product tag `great (c4c028f0-fec1-7512-83cd-c17537d1f502)` is not part of the request anymore, it will be removed from the association.
+
+[source,json]
+.Response: 200 - OK
+----
+{
+  "_entityName": "rstex11_Product",
+  "_instanceName": "Outback Power Remote Power System",
+  "id": "e1d586b4-aefb-2ee7-3b91-b07357b178ea",
+  "createdDate": "2021-03-05",
+  "createdBy": "admin",
+  "lastModifiedDate": "2021-03-05",
+  "price": 99.95,
+  "lastModifiedBy": "admin",
+  "name": "Outback Power Remote Power System",
+  "version": 2,
+  "tags": [
+    {
+      "_entityName": "rstex11_ProductTag",
+      "_instanceName": "shiny",
+      "id": "333f3a20-c47b-4bc9-ba34-a72d2d815695",
+      "name": "shiny" // <1>
+    },
+    {
+      "_entityName": "rstex11_ProductTag",
+      "_instanceName": "amazing",
+      "id": "d6ab132e-a0bd-a624-c6ad-cc544e83c584",
+      "name": "amazing" // <2>
+    }
+  ]
+}
+----
+<1> The `shiny` reference is still there, as it was part of the request
+<2> The `amazing` reference has been added, whereas the tag `great` is not part of the association anymore.
+
+[CAUTION]
```

Comment:
Opposite to my comment above, here NOTE seems to be enough, as nothing happens until user explicitly specifies null.